kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/KroxyliciousST.java

```diff
@@ -129,6 +129,47 @@ void kroxyWithReplicas(String namespace) {
         assertThat(result).withFailMessage("expected message have not been received!").contains(expectedMessage);
     }
 
+    @Test
+    void scaleUpKroxylicious(String namespace) {
+        int replicas = 2;
+        scaleKroxylicious(namespace, replicas, replicas + 1);
+    }
+
+    @Test
+    void scaleDownKroxylicious(String namespace) {
+        int replicas = 3;
+        scaleKroxylicious(namespace, replicas, replicas - 1);
+    }
+
+    private void scaleKroxylicious(String namespace, int replicas, int scaleTo) {
+        int numberOfMessages = 10;
+        String expectedMessage = MESSAGE + " - " + (numberOfMessages - 1);
+
+        // start Kroxylicious
+        LOGGER.info("Given Kroxylicious in {} namespace with {} replicas", namespace, replicas);
+        kroxylicious = new Kroxylicious(namespace);
+        kroxylicious.deployPortPerBrokerPlainWithNoFilters(clusterName, replicas);
+        String bootstrap = kroxylicious.getBootstrap();
+        int currentReplicas = kroxylicious.getNumberOfReplicas();
+        assertThat(currentReplicas).withFailMessage("unexpected current replicas").isEqualTo(replicas);
+
+        LOGGER.info("And KafkaTopic in {} namespace", namespace);
+        KafkaSteps.createTopic(namespace, topicName, bootstrap, 3, 2);
```

Comment:
Not something that needs addressed here, but it would be good to run all kafka commands through the proxy rather than some mixed mode.